### PR TITLE
Release 0.10.0

### PR DESCRIPTION
**Pull request by pitkant**
New release of sotkanet for CRAN.

### CODE REVIEW

**Review by antagomir — APPROVED** [review]
Seems good to me for now!

**Comment by Allaht2** [comment]
While working on interactive function for `fmi2` I noticed an error that caused `sotkanet_interactive` to fail, so I fixed it.

**Comment by antagomir** [comment]
Let me know if this is ready for CRAN (and merge should be done first I guess)

**Comment by Allaht2** [comment]
Everything should be ok.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,8 +1,8 @@
 Package: sotkanet
 Type: Package
 Title: Sotkanet Open Data Access and Analysis
-Version: 0.9.79
-Date: 2022-02-01
+Version: 0.10.0
```
Comment (antagomir) [inline DESCRIPTION:4]: Numbering must start from 1 -> make this 0.10.1

(Or should we already go for official release 1.1.1?

**Pull request merged**